### ISSUE TITLE
Basic Auth on frontend level

### ISSUE THREAD

**Issue by bastshoes** [containous/traefik#3673]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Using Traefik 1.7 RC2
Defined following labels for docker container:
```  
    - "traefik.frontend.auth.basic.usersfile=/shared/.htpasswd"
    - "traefik.frontend.auth.basic.removeHeader=true"
```
### What did you expect to see?

Successful authentication while getting access to defined frontend.

### What did you see instead?
Authentication failed.

Logs states that user not found:

```
Traefik          | time="2018-07-25T10:22:42Z" level=debug msg="User not found: admin"
Traefik          | time="2018-07-25T10:22:42Z" level=debug msg="Basic auth failed"
```
### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.6.5
Codename:     tetedemoine
Go version:   go1.10.3
Built:        2018-07-10_03:54:03PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
checkNewVersion = true


defaultEntryPoints = ["http", "https"]

# Force HTTPS
[entryPoints]
  [entryPoints.http]
  address = ":80"  
  [entryPoints.https]
  address = ":443"
    [entryPoints.https.tls]
      MinVersion = "VersionTLS12"
      [[entryPoints.https.tls.certificates]]
      certFile = "/shared/traefik_cert.pem"
      keyFile = "/shared/traefik_key.pem"
[Docker]
  endpoint = "unix:///var/run/docker.sock"
  domain = "homelab.loc"
  watch = true
  exposedbydefault = false 
  swarmMode = false
  network = 'proxy_backend'
```

### If applicable, please paste the log output in DEBUG level (`--logLevel=DEBUG` switch)

```
Traefik          | time="2018-07-25T10:22:07Z" level=debug msg="originLabelsmap[com.docker.compose.oneoff:False traefik.frontend.auth.basic.removeHeader:true com.docker.compose.container-number:1 com.docker.compose.version:1.21.2 org.label-schema.description:A modern reverse-proxy org.label-schema.vendor:Containous traefik.domain:homelab.loc com.docker.compose.config-hash:d07b05c054cd8a9b6fa88f74f9d97fcc2b59417bfa2ee79bd43b3c044530471b org.label-schema.url:https://traefik.io org.label-schema.version:v1.7.0-rc2 traefik.docker.network:proxy_backend traefik.frontend.auth.basic.usersfile:/shared/.htpasswd traefik.frontend.entryPoints:http,https org.label-schema.docker.schema-version:1.0 com.docker.compose.service:traefik org.label-schema.name:Traefik traefik.backend:traefik traefik.enable:true traefik.frontend.rule:Host:traefik.homelab.loc traefik.port:8080 com.docker.compose.project:homeautomation]"
Traefik          | time="2018-07-25T10:22:07Z" level=debug msg="allLabelsmap[:map[traefik.domain:homelab.loc traefik.frontend.entryPoints:http,https traefik.enable:true traefik.port:8080 traefik.frontend.auth.basic.removeHeader:true traefik.docker.network:proxy_backend traefik.frontend.auth.basic.usersfile:/shared/.htpasswd traefik.backend:traefik traefik.frontend.rule:Host:traefik.homelab.loc]]"

Traefik          | time="2018-07-25T10:22:07Z" level=debug msg="Configuration received from provider docker: {\"backends\":{\"backend-portainer\":{\"servers\":{\"server-Portainer-4c65ef31a2241e803616e2074cab862a\":{\"url\":\"http://172.18.0.3:9000\",\"weight\":1}},\"loadBalancer\":{\"method\":\"wrr\"}},\"backend-traefik\":{\"servers\":{\"server-Traefik-4f222dca3310205bcce6ca15b4b68224\":{\"url\":\"http://172.18.0.2:8080\",\"weight\":1}},\"loadBalancer\":{\"method\":\"wrr\"}}},\"frontends\":{\"frontend-Host-portainer-homelab-loc-0\":{\"entryPoints\":[\"http\",\"https\"],\"backend\":\"backend-portainer\",\"routes\":{\"route-frontend-Host-portainer-homelab-loc-0\":{\"rule\":\"Host:portainer.homelab.loc\"}},\"passHostHeader\":true,\"priority\":0,\"basicAuth\":null,\"auth\":{\"basic\":{\"users\":[\"user:$apr1$xxxxxxxxxxxxxxxxxxxx\"],\"removeHeader\":true}}},\"frontend-Host-traefik-homelab-loc-1\":{\"entryPoints\":[\"http\",\"https\"],\"backend\":\"backend-traefik\",\"routes\":{\"route-frontend-Host-traefik-homelab-loc-1\":{\"rule\":\"Host:traefik.homelab.loc\"}},\"passHostHeader\":true,\"priority\":0,\"basicAuth\":null,\"auth\":{\"basic\":{\"removeHeader\":true}}}}}"
Traefik          | time="2018-07-25T10:22:42Z" level=debug msg="User not found: admin"
Traefik          | time="2018-07-25T10:22:42Z" level=debug msg="Basic auth failed"
```

**Comment by traefiker** [comment]
Thanks for opening this issue!

We need further information to better understand the problem you're facing :thinking:

Could you please join us on our [Slack workspace](https://traefik.herokuapp.com) and reach out to us on the ([#support channel](https://traefik.slack.com/messages/C0CDT22PJ/))?

We're looking forward to talking to you there!

**Comment by bastshoes** [comment]
Issue solved. 
User file label is case sensitive. Instead of using
` - "traefik.frontend.auth.basic.usersfile=/shared/.htpasswd"` should be 
`- "traefik.frontend.auth.basic.usersFile=/shared/.htpasswd"`.

**Comment by traefiker** [comment]
Closed by #3730.
